app/src/main/java/com/sakaimobile/development/sakaiclient20/persistence/SakaiDatabase.java

```diff
@@ -35,7 +35,7 @@
         Attachment.class,
         Announcement.class,
         Resource.class
-}, version = 9)
+}, version = 13)
```

Comment:
Can we leave the version at 9 for now? I want to avoid incrementing the version unless we make schema changes for a major release, so the version number should only increment by 1 iff. we have a major release coming up. If you clear app data we won't get the runtime exception about schema changing.